tiffslide/tiffslide.py

```diff
@@ -843,6 +845,37 @@ def parse_generic_tiff(self) -> dict[str, Any]:
         md.update(self.collect_level_info(series0))
         return md
 
+    def parse_philips_tiff(self) -> dict[str, Any]:
+        """parse Philips tiff properties"""
+        md = self.parse_generic_tiff()
+        if self._tf.philips_metadata is None:
+            return md
+        philips_metadata = ElementTree.fromstring(self._tf.philips_metadata)
+        def get_first_attribute_with_name(
+            root: ElementTree.Element,
+            name: str
+        ) -> Optional[str]:
+            return next(
+                root.iterfind(f".//Attribute[@Name='{name}']")
+            ).text
+        md[PROPERTY_NAME_VENDOR] = get_first_attribute_with_name(
+            philips_metadata,
+            'DICOM_MANUFACTURER'
+        )
+        pixel_spacing_attribute = get_first_attribute_with_name(
+            philips_metadata,
+            'DICOM_PIXEL_SPACING'
+        )
+        if pixel_spacing_attribute is not None:
+            pixel_spacings = [
+                float(element.strip('"')) * 1000
+                for element in pixel_spacing_attribute.split(' ')
+            ]
+            mpp_x, mpp_y = pixel_spacings[0], pixel_spacings[1]
```

Comment:
@erikogabrielsson @ap-- i think x and y need to be reversed here. [openslide's website](https://openslide.org/formats/philips/) says the following:

```
openslide.mpp-x
    calculated as 1000 * philips.DICOM_PIXEL_SPACING[1]
openslide.mpp-y
    calculated as 1000 * philips.DICOM_PIXEL_SPACING[0] 
```

Reply:
You are correct. Per [DICOM specifications](https://dicom.innolitics.com/ciods/vl-whole-slide-microscopy-image/vl-whole-slide-microscopy-image-multi-frame-functional-groups/52009230/00289110/00280030) the first value is the row spacing and the second value the column spacing,